packages/rich-text-from-markdown/src/index.ts

```diff
@@ -67,14 +68,13 @@ const nodeContainerTypes = new Map([
   ['heading-4', 'block'],
   ['heading-5', 'block'],
   ['heading-6', 'block'],
-  ['listItem', 'block'],
+  ['list-item', 'block'],
```

Comment:
Can we use the constants exported by `@contentful/rich-text-types` instead of hard coding them?

Reply:
@mohamedturki done

Reply:
attaboy 👍